firefox-ios/Client/Utils/SponsoredContentFilterUtility.swift

```diff
@@ -15,7 +15,15 @@ struct SponsoredContentFilterUtility {
     }
 
     func filterSponsoredSites(from sites: [Site]) -> [Site] {
-        return sites.filter { !$0.url.contains(hideWithSearchParam) }
+        return sites.filter { item in
+            guard let url = URL(string: item.url) else {return true}
```

Comment:
```suggestion
            guard let url = URL(string: item.url) else { return true }
```

SwiftLint might catch this, but friendly style reminder if it doesn't.